build-scripts/gulp/translations.js

```diff
@@ -411,8 +410,10 @@ gulp.task(
 gulp.task(
   "build-translations",
   gulp.series(
-    "clean-translations",
-    "ensure-translations-build-dir",
+    gulp.parallel(
+      "fetch-nightly-translations",
```

Comment:
We don't wait for the completion of this task now, resulting in errors because of half downloaded files or no just no full translations yet

Reply:
I don't think that's the case.  I only put the fetch in parallel with the clean/ensure because they are happening in 2 different directories (`./translations` vs. `./build/translations`).  Both of those need to finish before going on to the next step in the main series.

Reply:
I tested it, and it did not wait for it, and ran into an error because of incorrect JSON

Reply:
Hmm... can you attach the log? I tested pretty thoroughly and never got an error.  Each of the CI jobs for this PR also ran it without errors.

Reply:
The supervisor job just failed with an error.  Is that the same as you got?  I don't think it's not waiting for the task.  Something else is wrong.

Reply:
So it looks like the function is not actually waiting on the tar extraction before returning, so my last line of code is the problem.  I'll fix it.